__tests__/endpoints.test.js

```diff
@@ -94,5 +94,39 @@ describe('NC news endpoint tests',()=>{
                 })
             })
         })
+        describe('GETting comments by article id',()=>{
+            test('/api/articles/:article_id/comments',()=>{
+                return request(app)
+                .get('/api/articles/1/comments')
+                .expect(200)
+                .then(({body})=>{
+                    body.comments.forEach(comment=>{
```

Comment:
The assertion missing here is the same as on your previous PR 😄  Let's check the array has the expected number of comments in it before running .forEach